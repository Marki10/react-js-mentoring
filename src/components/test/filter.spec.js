import React, { Component } from 'react';
import Filter from '../filter';
import { connect } from "react-redux";

const ReactTestRenderer = require('react-test-renderer');

describe('Filter', () => {
    it('Render filter', () => {
        const component = ReactTestRenderer.create(
            connect(<Filter />)
        );
        const testTemp = component.toJSON();
        expect(testTemp).toMatchSnapshot();
    })
});