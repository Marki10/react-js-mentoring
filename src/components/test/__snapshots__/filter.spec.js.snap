exports[`Filter Render filter 1`] = `null`;
